modules/views-dsl/src/androidMain/AndroidManifest.xml

```diff
@@ -1,5 +0,0 @@
-<!--
-  ~ Copyright 2019 Louis Cognault Ayeva Derman. Use of this source code is governed by the Apache 2.0 license.
-  -->
-
-<manifest package="splitties.views.dsl.core"/>
```

Comment:
This one is being replaced by `splitties.views.dsl`, which is a binary compatibility breakage.
I think it can be okay given the current state of Splitties, but is there a way to tell auto-manifest to have a special package name here, or not have it run for that module?

Reply:
Yes, I mentioned this in readme. Technically it is binary incompatible but is it really? The generated R class is only used internally. I am not sure if it counts breaking change if it is internal change.

Reply:
The `BuildConfig` class is in the package defined in the manifest, and it's exposed in inlined symbols are you can see in the changed files of this module in this PR.

That said, I don't think it'd affect the current users especially since I'm not aware of any library build upon this split.

Reply:
Easy way to override on a certain module is to keep the `AndroidManifest.xml` file for that module. I reverted this file along with breaking changes.